media/css/pebbles/components/_global-nav.scss

```diff
@@ -0,0 +1,594 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../pebbles/includes/lib';
+
+/* -------------------------------------------------------------------------- */
+// Common variables
+
+$color-firefox-light-orange: #ff9500;
+
+$mq-nav-tablet:     'screen and (min-width: 820px)';
+$mq-nav-desktop:    'screen and (min-width: 1050px)';
+
+/* -------------------------------------------------------------------------- */
+// Horizontal top navigation
+
+.moz-global-nav {
+    position: relative;
+    background: #fff;
+    overflow: hidden;
+
+    a:link,
+    a:visited {
+        color: #000;
+        font-weight: bold;
+        text-decoration: none;
+    }
+
+    a:hover,
+    a:focus,
+    a:active {
+        color: #000;
+        text-decoration: underline;
+        transition: color 0.1s ease-in-out;
+    }
+
+    // Three-dotted nenu button
+    .nav-button-menu {
+        -moz-appearance: none;
+        -webkit-appearance: none;
+        font-size: 0;
+        background: transparent;
+        border: none;
+        float: left;
+        height: 25px;
+        margin: 13px 0;
+        width: 50px;
+        padding: 0;
+
+        .p1,
+        .p2,
+        .p3 {
+            transition: transform .2s ease-in-out;
+        }
+
+        &:hover,
+        &:active {
+            .p1 {
+                transform: translateX(4px);
+            }
+
+            .p2 {
+                transform: translateX(-3px);
+            }
+
+            .p3 {
+                transform: translateX(4px);
+            }
+        }
+
+        &:focus {
+            outline: none;
```

Comment:
Yeah, I think I need to use a different styling for focus here. I'll change the `fill` color to `#646464` to match the wordmark.